packages/components/src/Tile/subcomponents/GenericTile/GenericTile.module.scss

```diff
@@ -0,0 +1,141 @@
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/shadow";
+@import "~@kaizen/design-tokens/sass/spacing";
+@import "../../../../styles/legacy/layout";
+@import "../../../../styles/animation";
+@import "../../../../styles/responsive";
+@import "./variables";
+
+$tilePaddingTop: $spacing-xl;
+
+.root {
+  perspective: 800px;
+}
+
+.tile {
+  width: $tileWidth;
+
+  [data-tile-grid] & {
+    width: auto;
+  }
+
+  height: $tileHeight;
+  position: relative;
+  box-shadow: $shadow-small-box-shadow;
+  background-color: $color-white;
+  transition: transform $ca-duration-slow $ca-default;
+  transform-style: preserve-3d;
+  border-radius: 7px;
+
+  @include ca-media-mobile {
+    width: 100%;
+    min-width: 226px;
+    max-width: $tileHeight;
+  }
+
+  @media (max-width: 480px) {
+    max-width: 100%;
+  }
+}
+
+.face {
+  position: absolute;
+  background-color: $color-white;
+  border-radius: 7px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  backface-visibility: hidden;
+
+  * {
+    backface-visibility: hidden;
+  }
+}
+
+.faceFront {
+  justify-content: space-between;
+  padding: $tilePaddingTop $spacing-md $spacing-md;
+}
+
+.faceBack {
+  padding-top: $tilePaddingTop;
+  transform: rotateY(180deg);
+  background-color: $color-blue-100;
+}
+
+.faceMoodPositive {
+  background-color: $color-green-100;
+}
+
+.faceMoodInformative {
+  background-color: $color-blue-100;
+}
+
+.faceMoodCautionary {
+  background-color: $color-yellow-100;
+}
+
+.faceMoodAssertive {
+  background-color: $color-orange-100;
+}
+
+.faceMoodNegative {
+  background-color: $color-red-100;
+}
+
+.faceMoodProminent {
+  background-color: $color-purple-100;
+}
+
+.isFlipped {
+  transform: rotateY(179deg);
+}
+
+.title {
+  text-align: center;
+  padding-bottom: $spacing-sm;
+  backface-visibility: hidden;
+}
+
+.actions {
+  display: flex;
+}
+
+.informationBtn {
+  @include ca-position($top: 0, $end: 0);
+
+  position: absolute;
+  color: rgba($color-purple-800-rgb, 0.5);
+}
+
+.information {
+  display: flex;
+  padding-bottom: $spacing-md;
+  padding-right: $spacing-md;
+  padding-left: $spacing-md;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
+  width: 100%;
+  height: 100%;
+  text-align: left;
+  border-radius: 0 0 7px 7px;
+
+  &[dir="rtl"],
+  [dir="rtl"] & {
+    padding-right: $spacing-md;
+    padding-left: $spacing-md;
```

Comment:
Heh, turns out this was pointless because the padding is the same on both sides 😅 

But for other cases you don't need to do this anyway when you could have just used `padding-inline`